### ISSUE TITLE
session.get_object returns object other than specified in path when target is missing

### ISSUE THREAD

**Issue by asmunds** [Volue-Public/energy-mesh-python#426]
I am doing the following:

```
with mesh_client.mesh_connection.create_session() as session:
    session.get_object(target='Model/Agder/Mesh.To_Areas/Norge.To_HydroProduction/Vannkraft.To_WaterCourses/Ulla-Førre')
```

which returns an object with path `Model/Agder/Ulla-Førre`

![image](https://github.com/Volue-Public/energy-mesh-python/assets/40594801/d4a50761-bacd-44d8-81e0-704359930d60)

As you can see, the returned object does not correspond to the requested target. This seems to me to be erroneous behavior. 

Note that the requested object does not exist. But still, I would expect an error message, not a wrong object.

**Comment by tnoczyns-volue** [comment]
This a bug. Today we have a planning session for the upcoming 2-week sprint. I'll let you know about the outcome and possible timeline.

**Comment by asmunds** [comment]
Great, thanks

**Comment by tnoczyns-volue** [comment]
Because of other urgent planned activities we didn't include this this issue into the new sprint. I'll inform you about the next planning session outcome in 2 weeks. Meanwhile please let me know about the urgency of this issue, so we can prioritize it better.

**Comment by tnoczyns-volue** [comment]
@asmunds, information from our planning session: the issue is included in our current sprint - starting today.

**Comment by tnoczyns-volue** [comment]
The issue was entirely on the Mesh server side. Now it is fixed and will be available in Mesh 2.14 which is planned to be released in June.

**Comment by asmunds** [comment]
Okay, thanks!
